### ISSUE TITLE
Update Readme

### ISSUE THREAD

**Issue by sofiapoh** [quaker-social-action/dte-website#206]
Update README to have more info on how to build the project and info about the project in general.

**Comment by gabrielperales** [comment]
Yeah, we should add some information about how to run it. I had to review your changes and I couldn't remember how to run it! So... we have to add `harp server` at someplace of the README file

**Comment by sofiapoh** [comment]
Same! Had to do some documentation digging and guessing...
